src/Controls/src/Core/Handlers/Items/iOS/ItemsViewLayout.cs

```diff
@@ -500,6 +500,10 @@ public override void FinalizeCollectionViewUpdates()
 			{
 				ForceScrollToLastItem(CollectionView, _itemsLayout);
 			}
+			else if (ItemsUpdatingScrollMode == ItemsUpdatingScrollMode.KeepItemsInView)
```

Comment:
There are changes in Android and iOS, happens the same on Windows?

Reply:
On Windows, `ItemsUpdatingScrollMode` correctly considers `OnItemsVectorChanged` so it is working fine